src/LibraryViewExtensionWebView2/Handlers/NodeItemDataProvider.cs

```diff
@@ -50,10 +52,11 @@ public NodeItemDataProvider(NodeSearchModel model) : base(false)
             this.model = model;
         }
 
-        public NodeItemDataProvider(NodeSearchModel model, IconResourceProvider iconProvider)
+        public NodeItemDataProvider(NodeSearchModel model, IconResourceProvider iconProvider, DynamoModel dynModel)
```

Comment:
Please let me know if this will produce an API Breaking Change since the method is public but we need to have the DynamoModel in order to pass the dynamoModel.LuceneSearchUtility to the Search method.

Reply:
I have concern over this and would like to discuss with @reddyashish and you on Monday